ios/FluentUI/Extensions/UIView+Extensions.swift

```diff
@@ -200,3 +200,17 @@ extension UIView {
         NSLayoutConstraint.contain(view: view, in: self, withInsets: insets, respectingSafeAreaInsets: respectsSafeAreaInsets)
     }
 }
+
+// MARK: - Pointer Interaction APIs check
+
+extension UIView {
+    
+    /// Some beta versions of iOS 13.4 and later don't include the pointer interaction APIs.
+    /// Checking the UIButton's isPointerInteractionEnabled property is a workaround that prevents a crash due to an
+    /// exception thrown while trying to add the UIPointerInteraction instance while running on those iOS versions:
+    /// "uncaught ObjC exception, reason: Invalid parameter not satisfying: interaction"
+    /// - Returns: true if Pointer Interaction APIs are available, otherwise false
+    func arePointerInteractionAPIsAvailable() -> Bool {
+        return UIButton.instancesRespond(to: Selector(("isPointerInteractionEnabled")))
```

Comment:
No practical impact, just a potential optimization that could help us detect and delete this when possible - thoughts on marking either the impl or the declaration #available with iOS 13.4 as well? That way we don't need to individually track such issues and can ensure it will get cleaned up as part of regularly scheduled combing for no longer relevant code when we do version bumps. Or is swift too smart and won't let us do that?

Reply:
actually yeah we could mark this func as available for 13.4 since all call sites currently have iOS 13.4 availability checks already out of necessity

Reply:
Good point. The func is now marked @available for iOS 13.4.